certificate-policy.md

```diff
@@ -1,419 +1,206 @@
 
-- to specify the Federal Public Device PKI Certificate Policy and requirements, and
-- to provide requirements for what each Certification Authority must address in its Certification Practice Statement
+- To specify the U.S. Federal Public Trust TLS PKI Certificate Policy and requirements, and
+- To provide requirements for what each Certification Authority shall address in its Certification Practice Statement
 
-This policy is for a hierarchical Public Key Infrastructure restricted to services operated by or on behalf of the U.S. Government and U.S. Federal entities.  The hierarchical PKI is referenced as the **Federal Public Device PKI** in this document.  
+This policy promotes automation to improve U.S. Federal Government efficiencies.  This policy also incorporates Certificate Transparency as a key component for publicly accessible and accountable services operated by the U.S. Federal Government.   
 
-This policy and requirements are applicable to all Certification Authorities within a chain of trust under the **US Federal Device Root CA**.  The requirements are to be flowed down from the Root Certification Authority through all Subordinate Certification Authorities.  
+This policy is applicable to all Certification Authorities within a chain of trust under the **U.S. Federal Public Trust TLS Root CA**.    
 
-The Federal Public Device PKI conforms to the Version 1.4.9 of the guidelines adopted by the Certification Authority/Browser Forum (“CA/Browser Forum”) when issuing publicly trusted certificates, including the Baseline Requirements for the Issuance and Management of Publicly Trusted Certificates (“Baseline Requirements”). This document is based on the CA/Browser Forum Baseline Requirements, which is licensed under the Creative Commons Attribution 4.0 International License. All additions and modifications made to create this document are in the public domain as works of the U.S. Government, and copyright and related rights in the work are waived.
+The terms and provisions of this certificate policy shall be interpreted under and governed by applicable Federal law.
 
-Additional documents related to the Federal Public Device PKI, such as Certification Practice Statements, Audits, and Subscriber Agreement(s) can be found at \<INSERT URL HERE>.
+## 1.1.1 Scope
+[TBD - statements for scoping to TLS server authentication certificates]
+[TBD - statements for enterprise trust versus public trust and definition]
 
-In accordance with RFC 3647, this CP includes all nine sections of the RFC 3647 framework and an additional addendum with the certificate profiles.    
+## 1.1.2 Compliance
+This Certificate Policy conforms to the current version of the Baseline Requirements for the Issuance and Management of Publicly-Trusted Certificates published at http://www.cabforum.org. In the event of any inconsistency between this document and those Baseline Requirements, those Baseline Requirements take precedence over this document.
```

Comment:
Noting for a future tiny fix: the URL for CABF is https://cabforum.org, not http://www.cabforum.org.